frontend/src/app/application-dashboard/application-dashboard.module.ts

```diff
@@ -29,10 +30,12 @@ const DashboardRoutes: Routes = [
     PageListComponent,
     PageComponent,
     CsiValueComponent,
-    CsiGraphComponent
+    CsiGraphComponent,
+    PageMetricComponent
   ],
   exports: [
-    RouterModule
+    RouterModule,
+    PageMetricComponent
```

Comment:
please do not export the PageMetricComponent (unless you need to use it in another module without a dedicated import statement)

Reply:
done.